addons/crm/models/crm_lead.py

```diff
@@ -1374,6 +1392,16 @@ def _pls_get_naive_bayes_probabilities(self, batch_mode=False):
         save_team_id = None
         p_won, p_lost = 1, 1
         for lead_id, lead_values in leads_values_dict.items():
+            # if stage_id is null, return 0 and bypass computation
+            lead_fields = [values[0] for values in list(lead_values.get('values', {}))]
```

Comment:
list(dict) -> get it keys. Is it really what you want ? Seems weird.

If it is the case, just to list (dict.keys()), it is easier to understand